<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片水印</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/m-水印.png</normaloff>images/m-水印.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="outfileRoadText"/>
    </item>
    <item>
     <widget class="QPushButton" name="outfileRoadBtn">
      <property name="text">
       <string>选择输出目录</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/open.png</normaloff>images/open.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="compressBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加水印</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/水印.png</normaloff>images/水印.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>输入水印文字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="watermarkText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>透明度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lut">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>水印位置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="siteBox">
      <item>
       <property name="text">
        <string>左上</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>正中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>右下</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>字体大小</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fontSizetext">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>20</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="fileRoadSelect"/>
    </item>
    <item>
     <widget class="QComboBox" name="videoTypeBox">
      <item>
       <property name="text">
        <string>all</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.jpg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.png</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="imageSelectBtn">
      <property name="text">
       <string>选择文件</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/素材.png</normaloff>images/素材.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
